server/app/views/components/ProgramCardFactory.java

```diff
@@ -65,9 +70,19 @@ public DivTag renderCard(ProgramCardData cardData) {
                                 ReferenceClasses.ADMIN_PROGRAM_CARD_TITLE,
                                 "text-black",
                                 "font-bold",
-                                "text-xl"),
+                                "text-xl"))
+                    .with(
                         p(programDescriptionText)
-                            .withClasses("line-clamp-2", "text-gray-700", "text-base")),
+                            .withClasses("line-clamp-2", "text-gray-700", "text-base"))
+                    .condWith(
+                        featureFlags.isIntakeFormEnabled(request)
```

Comment:
Can we pull this check out into its own method, like this:
https://github.com/civiform/civiform/blob/5fbdecbab2ec80ff185857ecdd985eabc65bc1bc/server/app/views/components/QuestionBank.java#L258-L262

I think that will make this a little easier to scan quickly